### PR TITLE
Upgrade orval and start using urlEncodeParameters

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

File: src/oasis-nexus/yarn.lock [hunk 1]
```diff
@@ -2009,6 +2502,59 @@ validator@^13.7.0:
...
+  resolved "https://registry.yarnpkg.com/vite/-/vite-5.0.8.tgz#070383dcb4d4f12adde9c119b8cca0f40bf906e9"
+  integrity sha512-jYMALd8aeqR3yS9xlHd0OzQJndS9fH5ylVgWdB+pxTwxLKdO1pgC5Dlb398BUxpfaBxa4M9oT7j1g503Gaj5IQ==
+  dependencies:
+    esbuild "^0.19.3"
+    postcss "^8.4.32"
+    rollup "^4.2.0"
+  optionalDependencies:
+    fsevents "~2.3.3"
+
+vitest@^0.34.6:
```
Comment (buberdds) [inline src/oasis-nexus/yarn.lock:2528]: I've left a comment in Renovate PR but their last deploy adds tons of orval dev deps to our node modules (vite, rollup, esbuild etc).

Comment (lukaw3d) [reply src/oasis-nexus/yarn.lock:2528]: good catch. did you comment anywhere upstream?

Comment (buberdds) [reply src/oasis-nexus/yarn.lock:2528]: nope

Comment (lukaw3d) [reply src/oasis-nexus/yarn.lock]: Fixed in https://github.com/anymaniax/orval/pull/1155

Comment (csillag) [reply src/oasis-nexus/yarn.lock]: Apparently, merging the fix just missed the release of v6.24.0 :(

**Review by buberdds — APPROVED** [review]

**Pull request merged**